patterns/terraform/main.tf

```diff
@@ -91,37 +111,29 @@ module "eks" {
 
 module "eks_blueprints_addons" {
   source  = "aws-ia/eks-blueprints-addons/aws"
-  version = "~> 1.0.0"
+  version = "~> 1.16"
 
   cluster_name      = module.eks.cluster_name
   cluster_endpoint  = module.eks.cluster_endpoint
   cluster_version   = module.eks.cluster_version
   oidc_provider_arn = module.eks.oidc_provider_arn
 
-  eks_addons = {
-    coredns = {
-      timeouts = {
-        create = "25m"
-        delete = "10m"
-      }
-    }
-    vpc-cni    = {}
-    kube-proxy = {}
-  }
-
   # Add-ons
   enable_aws_load_balancer_controller = true
   enable_metrics_server               = true
 
   tags = local.tags
+
+  depends_on = [module.eks]
 }
 
 ################################################################################
 # ACK Addons
 ################################################################################
 
 module "eks_ack_addons" {
-  source = "../../"
+  source  = "aws-ia/eks-ack-addons/aws"
+  version = "2.2.0"
```

Comment:
This is done relatively for a reason so you can test the latest changes. Please leave it as is.